### PR TITLE
[Enhancement](ranger) Disable some permission operations when Ranger or LDAP are enabled

### PR DESCRIPTION
**Pull request by yongjinhou**
## Proposed changes

Issue Number: close ##32536

Disable some permission operations when Ranger or LDAP are enabled.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by yongjinhou** [comment]
run build all

**Comment by yongjinhou** [comment]
run build all

File: fe/fe-core/src/main/java/org/apache/doris/analysis/CreateRoleStmt.java [hunk 1]
```diff
@@ -50,6 +52,11 @@ public String getRole() {
     @Override
     public void analyze(Analyzer analyzer) throws UserException {
         super.analyze(analyzer);
+
+        if (Config.access_controller_type.equalsIgnoreCase("ranger-doris")) {
```
Comment (chenlinzhong) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/CreateRoleStmt.java:56]: better use const variable to replace "ranger-doris"

Comment (yongjinhou) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/CreateRoleStmt.java:56]: 跑完流水线看下有没有其他问题，一起改下吧

**Comment by yongjinhou** [comment]
run build all

**Comment by yongjinhou** [comment]
run buildall

**Comment by yongjinhou** [comment]
run compile

**Comment by yongjinhou** [comment]
run buildall

**Comment by yongjinhou** [comment]
run buildall

**Comment by yongjinhou** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Comment by LompleZ** [comment]
LGTM

**Review by LompleZ — APPROVED** [review]
LGTM

**Comment by yongjinhou** [comment]
run Need_2_Approval

**Review by xy720 — APPROVED** [review]
LGTM

**Pull request merged**